src/patch-events.js

```diff
@@ -128,7 +134,7 @@ function retarget(refNode, path) {
   }
 }
 
-let eventMixin = {
+let EventPatches = {
```

Comment:
[`mixinComposedFlag`](https://github.com/webcomponents/shadydom/pull/298/files#diff-6485b9ef567138f24ff200fdb1d123c9R214) below could do `Object.setPrototypeOf(klazz, Base)` to chain the constructors, rather than copy values over.

Reply:
Done.